influence_benchmark/backend/hf_backend.py

```diff
@@ -50,40 +50,22 @@ def __init__(
         assert self.device is not None, "Device must be specified"
         self.tokenizer = AutoTokenizer.from_pretrained(model_name, padding_side="left")
         self.lora_active = False
-        if inference_quantization == "8-bit" or inference_quantization == "4-bit":
```

Comment:
You seem to have just removed support for quantization completely? We need this for 70B

Reply:
Ah, ok will add back in